### PR TITLE
[#16] Implement UI for survey list item on Home screen

### PR DESCRIPTION
**Pull request by chornerman**
#16

## What happened 👀

Implement UI for survey list item on Home screen

## Insight 📝

- [x] Create a survey list item widget
- [x] Add resources used in the survey list item widget such as string, text style, color, and icon
- [x] Restructure `HomePage` and `HomeHeaderWidget`

### CODE REVIEW

**Review by markgravity — APPROVED** [review]
LGTM

File: lib/page/home/widget/home_surveys_item_widget.dart [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      child: Padding(
+        padding: const EdgeInsets.all(Dimens.space20),
+        child: Column(
+          children: [
+            Expanded(
+              child: const SizedBox.shrink(),
+            ),
+            Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
```
Comment (luongvo) [inline lib/page/home/widget/home_surveys_item_widget.dart:39]: Why do we need to use 2 Column widgets here? Not sure what's your purpose here.

Can we optimize to use only 1?

**Review by luongvo — COMMENTED** [review]
rest lgtm

File: lib/page/home/widget/home_surveys_item_widget.dart (hunk 1 above)
Comment (chornerman) [reply lib/page/home/widget/home_surveys_item_widget.dart]: Nice catch, updated 🚀

**Review by luongvo — APPROVED** [review]

**Pull request merged**